### PR TITLE
Test2

### PR DESCRIPTION
**Pull request by sachincv2**
This is my pull request for code review

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -30,6 +32,7 @@ gem 'font-awesome-sass', '~> 5.13.0'
 #used to get icons and styles
 
 gem 'devise'
+#devise helps authenticating users with its built in methods and provides helpers to get currentuser and his details taken during registrations
```
Comment (gagan-lokesh) [inline Gemfile:33]: Add this comment before defining the gem

**Review by gagan-lokesh — CHANGES_REQUESTED** [review]
This looks good. Make sure to test these changes and confirm whether everything is working as expected.

File: Gemfile (hunk 1 above)
Comment (sachincv2) [reply Gemfile:33]: i have changed this

**Comment by sachincv2** [comment]
> This looks good. Make sure to test these changes and confirm whether everything is working as expected.

Hey  i have tested all the features implemented . It is working well

**Comment by gagan-lokesh** [comment]
@sachincv2 Did you test for responsiveness?

**Comment by sachincv2** [comment]
> @sachincv2 Did you test for responsiveness?

Yes i have tested it

**Review by gagan-lokesh — APPROVED** [review]

**Pull request merged**